draft-ietf-quic-recovery.md

```diff
@@ -1000,6 +999,17 @@ The recovery period limits congestion window reduction to once per round trip.
 During recovery, the congestion window remains unchanged irrespective of new
 losses or increases in the ECN-CE counter.
 
+## Loss of protected packets during the handshake
+
+0RTT and 1RTT packets sent prior to handshake completion can arrive before
```

Comment:
I think that you need to identify this as early RTT packets that are sent by an endpoint with new keys.  Also, expand on that: Handshake and 0-RTT from the client, and 1-RTT from either endpoint.